source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/include/ACOAdaptiveCard.h

```diff
@@ -0,0 +1 @@
+../ACOAdaptiveCard.h
```

Comment:
just curious here -- is there no cleaner way of doing this?

Reply:
Unfortunately, I couldn't find one.